src/uct/sm/scopy/cma/cma_iface.c

```diff
@@ -74,16 +74,28 @@ uct_cma_iface_is_reachable(const uct_iface_h tl_iface,
                            const uct_iface_addr_t *tl_iface_addr)
 {
     ucs_cma_iface_ext_device_addr_t *iface_addr = (void*)tl_iface_addr;
+    int dummy                                   = 0;
+    struct iovec iov                            = {
+        .iov_base = &dummy,
```

Comment:
minor: iov could point to itself (instead of adding int dummy)

Reply:
fixed